# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnomemeeting/openldap23.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur gnomemeeting-1.2.2.orig/src/ils.cpp gnomemeeting-1.2.2/src/ils.cpp
--- gnomemeeting-1.2.2.orig/src/ils.cpp	2004-10-24 23:07:04.000000000 +0300
+++ gnomemeeting-1.2.2/src/ils.cpp	2005-11-02 11:59:44.000000000 +0200
@@ -47,8 +47,6 @@
 #include "stock-icons.h"
 #include "dialog.h"
 
-#include <ldap.h>
-
 #include "../pixmaps/inlines.h"
 
 
diff -Nur gnomemeeting-1.2.2.orig/src/ils.h gnomemeeting-1.2.2/src/ils.h
--- gnomemeeting-1.2.2.orig/src/ils.h	2004-09-10 12:44:23.000000000 +0300
+++ gnomemeeting-1.2.2/src/ils.h	2005-11-02 11:59:29.000000000 +0200
@@ -39,6 +39,8 @@
 #ifndef _ILS_H_
 #define _ILS_H_
 
+#define LDAP_DEPRECATED 1
+
 #include "common.h"
 #include "xdap.h"
 
